src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_performing_slr_with_regular_exception.cs

```diff
@@ -26,7 +26,7 @@ public class Context : ScenarioContext
                     .When(session => session.SendLocal(new MessageToBeRetried()))
                     .DoNotFailOnErrorMessages())
                 .Done(c => c.SlrChecksum != default(byte))
-                .Run();
+                .Run(TimeSpan.FromSeconds(120));
```

Comment:
We already have a max time, how about we let the transport change the default via `ConfigureXYZTransport` instead?